### PR TITLE
Enable android sdk test for native code CI process

### PR DESCRIPTION
**Pull request by zhangyihui1**

### CODE REVIEW

File: jenkins/android.jenkinsfile [hunk 1]
```diff
@@ -26,16 +26,67 @@ pipeline {
...
+                python ${androidSdkTestScript} --git-branch master --git-change-id null --git-commit ${GIT_COMMIT} --owner open-webrtc-toolkit --repo owt-client-native \
+                --remote-dependence-host ${env.remoteDependenceHost} --remote-dependence-host-username ${env.remoteDependenceUserName} \
+                --remote-dependence-host-password ${env.remoteDependencePassword} --remote-dependence-path ${env.remoteDependeceDebugPath}  \
+                --instrumentation ${env.androidSdkCaselistPath} --unit --p2p-server http://${env.p2pServer}:8095 --conference-server-http http://${env.mcuServer}:3001"
+            }
+        }
     }
     post {
         always {
-            echo "One way or another, I have finished"
```
Comment (yanbin7) [inline jenkins/android.jenkinsfile]: remove' One way or another '

File: jenkins/android.jenkinsfile [hunk 2]
```diff
@@ -26,16 +26,67 @@ pipeline {
...
+                    label "${env.androidRunCaseServer}"
+                    customWorkspace "${env.androidSourcePath}"
+                }
+            }
+            steps {
+                sh "docker run -v /dev/bus/usb:/dev/bus/usb -v ${env.androidCILogPath}:/home/logs --network host --privileged --rm  ${env.androidSdkImage} \
+                python ${androidSdkTestScript} --git-branch master --git-change-id null --git-commit ${GIT_COMMIT} --owner open-webrtc-toolkit --repo owt-client-native \
+                --remote-dependence-host ${env.remoteDependenceHost} --remote-dependence-host-username ${env.remoteDependenceUserName} \
+                --remote-dependence-host-password ${env.remoteDependencePassword} --remote-dependence-path ${env.remoteDependeceDebugPath}  \
+                --instrumentation ${env.androidSdkCaselistPath} --unit --p2p-server http://${env.p2pServer}:8095 --conference-server-http http://${env.mcuServer}:3001"
```
Comment (yanbin7) [inline jenkins/android.jenkinsfile]: Don't hardcode port e.g 8095 and 3001 , update port don't need update jenkinfile

File: jenkins/android.jenkinsfile (hunk 1 above)
Comment (zhangyihui1) [reply jenkins/android.jenkinsfile]: done

File: jenkins/android.jenkinsfile (hunk 2 above)
Comment (zhangyihui1) [reply jenkins/android.jenkinsfile]: done

**Pull request merged**